flink-table/flink-table-common/src/test/java/org/apache/flink/table/types/inference/TypeStrategiesTestBase.java

```diff
@@ -154,6 +160,11 @@ public TestSpec expectErrorMessage(String expectedErrorMessage) {
             return this;
         }
 
+        public TestSpec compareConversionClass() {
```

Comment:
This appears to be unused.

Reply:
Yes, I added this to the test framework in case anyone would like to actually compare based on the old behaviour. Not sure, what could be the use case, but wanted to leave that possibility.